src/api/lib/tasks/dev/test_data/maintenance.rb

```diff
@@ -174,35 +144,45 @@ def create_maintenance_setup
+             target_project_name: maintenance_project).tap do |bs_request|
+        puts "* Request #{bs_request.number} with #{bs_request.bs_request_actions.size} maintenance incident actions has been created."
+      end
 
       # Create the third incident request (many actions; with patchinfo)
-      bs_request = create_request_with_incident_actions(source_project_name: update_project_branch.name,
-                                                        target_project_name: maintenance_project,
-                                                        source_package_names: ['cacti', 'cacti-spine'],
-                                                        target_releaseproject_names: [update_project1.name, update_project2.name],
-                                                        patchinfo: true)
-      # Accept the last incident request
-      admin = User.get_default_admin
-      User.session = admin
-      bs_request.change_state(newstate: 'accepted', force: true, user: admin.login, comment: 'Accepted by admin')
+      create(:bs_request_with_maintenance_incident_actions,
+             :with_patchinfo,
+             creator: iggy,
+             description: 'Request with incident actions',
+             source_package_names: ['cacti', 'cacti-spine'],
+             target_releaseproject_names: [update_project1.name, update_project2.name],
+             source_project_name: update_project_branch.name,
+             target_project_name: maintenance_project).tap do |bs_request|
+        puts "* Request #{bs_request.number} with #{bs_request.bs_request_actions.size} maintenance incident actions has been created."
+        # Accept the last incident request
+        admin = User.get_default_admin
+        admin.run_as do
+          bs_request.change_state(newstate: 'accepted', force: true, user: admin.login, comment: 'Accepted by admin')
+        end
+      end
 
       # Open a request from a package that is not branched, but developed on an "official" project.
       # Simulate `osc maintenancerequest servers apache2 openSUSE:Leap:15.4:Update`
-      create_request_with_incident_actions(source_project_name: 'servers',
-                                           target_project_name: maintenance_project.name,
-                                           source_package_names: ['apache2'],
-                                           target_releaseproject_names: [update_project1.name],
-                                           patchinfo: false)
-
-      # Create maintenance release request that asks for releasing the changes on openSUSE:Maintenance:0 to openSUSE:*:Update.
-      create_request_with_maintenance_release_actions(source_project_name: 'openSUSE:Maintenance:0',
-                                                      package_names: ['cacti', 'cacti-spine'],
-                                                      target_project_names: [update_project1.name, update_project2.name])
```

Comment:
I think you removed the maintenance release part by accident when rebasing on master. Can you have a look?

Reply:
True, it got wiped out during the rebase... I'll bring it back

Reply:
That, in fact, is a factory now...

Reply:
Fixed